demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -241,3 +242,14 @@ def is_root_connected_to_all_tasks(self):  # type: () -> bool
                 return False
 
         return tasks_bucket.issubset(next_tasks_bucket)
+
+    def is_valid_deprecated_playbook(self) -> bool:
+        is_valid = True
+        hidden = self.current_file.get('hidden', False)
```

Comment:
change to `is_hidden`, and change in the `script.py` file to `is_deprecated` please